src/main/java/core/basesyntax/stategy/handler/PurchaseOperation.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.stategy.handler;
```

Comment:
The package name should be 'strategy' instead of 'stategy' to correctly reflect the pattern's name and to follow Java naming conventions.